.github/workflows/ui-ci.yml

```diff
@@ -0,0 +1,280 @@
+          GH_REPO: ${{ github.event.repository.name }}
+          GITHUB_SHA: ${{ github.sha }}
+        with:
+          entrypoint: /bin/sh
+          args: >
+            -c "
+            . $VIRTUAL_ENV/bin/activate &&
+            cd $ACTIONS_PATH/whitesource-policy-violation-checker &&
+            python whitesource_policy_violation_checker.py
+            "
+
+      - name: Run WhiteSource Vulnerability Checker
+        id: whitesource-vulnerability-checker
+        if: github.event_name == 'push' && github.ref_name == github.event.repository.default_branch
+        uses: docker://ghcr.io/solacedev/maas-build-actions:latest
+        continue-on-error: true
+        env:
+          WS_API_KEY: ${{ secrets.WHITESOURCE_API_KEY }}
+          WS_PRODUCT_NAME: "solaceai"
+          WS_PROJECT_NAME: "solace-agent-mesh-ui"
+          AWS_ACCESS_KEY_ID: ${{ secrets.SAM_AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.SAM_AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
+          WS_JIRA_CHECK: "True"
+          JIRA_ONLY: "True"
+          STATUS_CHECK: "True"
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GH_ORG: ${{ github.repository_owner }}
+          GH_REPO: ${{ github.event.repository.name }}
+          GITHUB_SHA: ${{ github.sha }}
+        with:
+          entrypoint: /bin/sh
+          args: >
+            -c "
+            . $VIRTUAL_ENV/bin/activate &&
+            cd $ACTIONS_PATH/whitesource-vulnerability-checker &&
+            python whitesource_vulnerability_checker.py
+            "
+
+  bump-version:
+    needs: ui-build-and-test
+    name: "Bump UI Version"
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && github.ref_name == github.event.repository.default_branch
+    outputs:
+      new-tag: ${{ steps.bump.outputs.newTag }}
+    defaults:
+      run:
+        working-directory: client/webui/frontend
+    steps:
+      - name: "Checkout source code"
+        uses: "actions/checkout@v4"
+        with:
+          ref: ${{ github.ref }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          fetch-depth: 0
+
+      - name: "Current package.json"
+        run: cat ./package.json
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - name: "Automated Version Bump"
+        id: bump
+        uses: "phips28/gh-action-bump-version@master"
+        with:
+          minor-wording: "feat,minor"
+          major-wording: "BREAKING CHANGE,major"
+          patch-wording: "fix,patch,docs,style,refactor,perf,test,build,ci,chore,revert"
+          tag-prefix: "ui-v"
+          commit-message: "ci(ui): bump version to {{version}} [skip ci]"
+          skip-push: "true"
+          default: "patch" # default to patch bump if no bumps included in commit messages
+          bump-policy: "ignore" # always bump regardless of whether bumps included in commit messages
```

Comment:
Automatically bump with patch version when conventional commit format is not followed